turbopuffer/backend.py

```diff
@@ -107,18 +107,32 @@ def make_api_request(self,
 
                 server_timing_str = response.headers.get('Server-Timing', '')
                 if len(server_timing_str) > 0:
-                    match_cache_hit_ratio = re.match(r'.*cache_hit_ratio;ratio=([\d\.]+)', server_timing_str)
-                    if match_cache_hit_ratio:
+                    match_cache = re.match(r'.*cache;hit_ratio=([\d\.]+);temperature=(\w+)', server_timing_str)
+                    if match_cache:
                         try:
-                            performance['cache_hit_ratio'] = float(match_cache_hit_ratio.group(1))
+                            performance['cache_hit_ratio'] = float(match_cache.group(1))
+                            performance['cache_temperature'] = match_cache.group(2)
+                        except ValueError:
+                            pass
+                    match_exhaustive_search = re.match(r'.*exhaustive_search;count=([\d\.]+)', server_timing_str)
+                    if match_exhaustive_search:
+                        try:
+                            performance['exhaustive_search_count'] = int(match_exhaustive_search.group(1))
                         except ValueError:
                             pass
```

Comment:
Seems like we match on exhaustive search count twice?

Reply:
ahhh yes, meant to remove that one